### PR TITLE
Ecf 113847619 config token valid period

### PR DESCRIPTION
**Pull request by ericcf**
Allow for configurable valid period for config tokens. Fix rubocop offenses. Lock active_model_serializers version.

### CODE REVIEW

**Comment by ericcf** [comment]
FYI the security issues are false positives because the `Gemfile.lock` is not included in the repo intentionally.

File: app/controllers/token_auth/api/payloads_controller.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def find_token!
 
         return if @authentication_token
 
-        fail TokenAuth::Concerns::ApiResources::Api::Unauthorized
+        raise TokenAuth::Concerns::ApiResources::Api::Unauthorized
```
Comment (eschlange) [inline app/controllers/token_auth/api/payloads_controller.rb:56]: Is there a benefit to using raise instead of fail?

**Comment by eschlange** [comment]
Looks good!

File: app/controllers/token_auth/api/payloads_controller.rb (hunk 1 above)
Comment (ericcf) [inline app/controllers/token_auth/api/payloads_controller.rb:56]: Just a RuboCop reversal that I didn't feel like fighting with. They are identical methods from what I can tell.

Comment (michaelwehrley) [inline app/controllers/token_auth/api/payloads_controller.rb:56]: Slight nuance reported [here](http://stackoverflow.com/questions/31937632/fail-vs-raise-in-ruby-should-we-really-believe-the-style-guide):

> use 'raise' for exceptions to be caught, and 'fail' for serious errors which are not meant to be handled

**Pull request merged**